### PR TITLE
Worker run method

### PR DESCRIPTION
**Pull request by andrewgy8**
### :tophat: What?

Add a shortcut method for calling setup, start, and sleep. 

### :thinking: Why?

Make it easier to start a worker.

### :link: Related issue

#114

### CODE REVIEW

**Comment by andrewgy8** [comment]
Yeah I will do that. I just didnt want the diff to get all cluttered.

File: rele/worker.py [hunk 1]
```diff
@@ -48,6 +49,15 @@ def start(self):
                 )
             )
 
+    def run(self, wait=60):
```
Comment (tobami) [inline rele/worker.py]: A `run` that ends after a given number of seconds does not make things much easier. I would instead implement a `run_forever(sleep_interval=2)`, for example.

Comment (hobochild) [reply rele/worker.py]: Yea this would be great, maybe something like: 

```

Comment (andrewgy8) [reply rele/worker.py]: By popular demand, I added the `run_forever` method. 👍

**Pull request merged**